.github/workflows/quality_checks.yml

```diff
@@ -45,3 +47,9 @@ jobs:
 
       - name: Run Unit Tests
         run: make test
+
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does GITHUB_TOKEN also need adding alongside SONAR_TOKEN at the top of the file, and passing-in in the same manner from the calls to quality_checks?

Reply:
no - GITHUB_TOKEN is automatically created when a github action runs and is available to called workflows - see https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/reusing-workflows

Reply:
`secrets.GITHUB_TOKEN` is available globally afaik